dataloaders/val/RobotCarDataset.py

```diff
@@ -9,16 +9,16 @@
 # I hardcoded the image names and ground truth for faster evaluation
 # performance is exactly the same as if you use VPR-Bench.
 
-DATASET_ROOT = "/home/USER/work/datasets/robotcar/"
```

Comment:
All Paths must be taken taken from utils/config.py
Please make the code more by doing this task where you feel free.